cypress/integration/work-order/create/create.spec.js

```diff
@@ -1759,6 +1766,313 @@ describe('Raise repair form', () => {
       )
     })
   })
+
+  context('When a user removes a contact detail', () => {
+    const contactId = '5b718087-33cd-746d-8f91-e7cec2299a73'
+    const personId = 'cfe9c99f-7636-e23d-f08a-f36083e57d5d'
+
+    beforeEach(() => {
+      cy.loginWithAgentRole()
+    })
+
+    it('shows a confirmation message when clicking on the remove button', () => {
+      cy.visit('/properties/00012345/raise-repair/new')
+
+      cy.wait([
+        '@propertyRequest',
+        '@contactDetailsRequest',
+        '@sorPrioritiesRequest',
+        '@tradesRequest',
+      ])
+
+      // remove first tenant
+      cy.contains('Remove').first().click()
+
+      // assert modal open
+      cy.contains(
+        `Are you sure you want to permanently remove mainNumber: 00000111111 from Mark Gardner?`
```

Comment:
Any reason we can't remove 'permanently' ? It slightly implies we won't be able to re-add them in the future

Reply:
I agree with your point. I can remove it